sentences/nb/cover_HassTurnOff.yaml

```
@@ -2,20 +2,20 @@ language: nb
 intents:
   HassTurnOff:
     data:
+      - sentences:
+          - "<lukk> <navn> [<i_på> <område>]"
+        requires_context:
+          domain: cover
+        response: cover
       - sentences:
           - "<lukk> <garasje>"
         slots:
           domain: cover
           device_class: garage
         response: cover_device_class
       - sentences:
-          - "<lukk> [<vindu>|<gardin>|<rullegardin>|<skodde>] <i_på> <område>"
-          - "<lukk> <område>[s][ ][<vindu>|<gardin>|<rullegardin>|<skodde>]"
+          - "<lukk> [the] {cover_classes:device_class} <i_på> <område>"
```

Another `the` I missed yesterday.
```suggestion
          - "<lukk> {cover_classes:device_class} <i_på> <område>"
```